src/zkevm_specs/evm/table.py

```diff
@@ -212,6 +218,27 @@ class CallContextFieldTag(IntEnum):
     StateWriteCounter = auto()
 
 
+class TxLogFieldTag(IntEnum):
```

Comment:
@DreamWuGit I want to share some alternative ideas for the usage of the rw_table.  This is different from your design but maybe you find something interesting.  Feel free to use some of the ideas or none at all!


| 0 *rwc*  | 1 *isWrite* | 2 *Key0 (Tag)* | 3 *Key1* | 4 *Key2* | 5 *Key3*    | 6 *Key4*    | 7 *Value0* | 8 *Value1* | 9 *Aux0* | 10 *Aux1*  |
| -------- | ----------- | -------------- | -------- | -------- | ----------- | ----------- | ---------  | ---------- | -------- | ---------- |
|          |             |                |          |          |             |             |            |            |          |            |
|          |             |                |          |          | *TxLogTag*  |             |            |            |          |            |
| $counter | true        | TxLog          | $logID   |          | Address     |             | $value     |            |          |            |
| $counter | true        | TxLog          | $logID   |          | TxID        |             | $value     |            |          |            |
| $counter | true        | TxLog          | $logID   |          | Topic0      |             | $value     |            |          |            |
| $counter | true        | TxLog          | $logID   |          | Topic1      |             | $value     |            |          |            |
| $counter | true        | TxLog          | $logID   |          | Topic2      |             | $value     |            |          |            |
| $counter | true        | TxLog          | $logID   |          | Topic3      |             | $value     |            |          |            |
|          |             |                |          |          |             |             |            |            |          |            |
| $counter | true        | TxLogData      | $logID   |          | $byteIndex  |             | $value     |            |          |            |

Here are the changes:
1. Added TxID tag
2. Instead of specifying the topic number using an aux column, use 4 different tags for each topic
3. Add a new RwTag for the content of the log, this way it has the same shape as the Memory rows.

Reply:
:)   thanks,  for topics ,  the topic count  is not fixed, from 0 to 4 (from `log0` to `log4`),  topics are reading from stack,  using fixed 4 will cause different behavior with stack operations.   I see your `logID ` as a synonym for `log_index`.

Reply:
> :) thanks, for topics , the topic count is not fixed, from 0 to 4 (from `log0` to `log4`), topics are reading from stack, using fixed 4 will cause different behavior with stack operations.

In the proposed table I was thinking that only the required Topic number would be used.  Like this:
```python 
    for i in range(4):
        if i < topic_count:
            topic_is_zeros[i] = 0
            topic = instruction.stack_pop()
            instruction.constrain_equal(topic, instruction.tx_log_lookup(TxLogFieldTag.Topic0 + i))
```
But maybe that complicates other things.

> I see your `logID ` as a synonym for `log_index`.  

Yes, exactly! I used `logID` in the example as an equivalent, because we're using `*ID` everywhere (like TxID, CallID, etc.)

Reply:
> This tag is not defined yet in https://github.com/appliedzkp/zkevm-specs/blob/master/specs/tables.md.
> 
> I would appreciate this PR also modifies this file to reflect the existance of this Tag and it's usage.

updated in [cb6f40b](https://github.com/appliedzkp/zkevm-specs/pull/125/commits/cb6f40ba0eaf2db317a57a885739aeedc8e9985e) thanks!